webapp/app/js/controllers/main.js

```diff
@@ -109,6 +110,13 @@ treeherder.controller('MainCtrl', [
                 $rootScope.$emit(thEvents.selectPreviousUnclassifiedFailure);
             });
 
+            // Shortcut: retrigger selected job
+            Mousetrap.bind('r', function() {
+                $scope.$evalAsync(
+                    $rootScope.$emit(thEvents.jobRetrigger, $rootScope.selectedJob)
```

Comment:
For the 'space' and 'c' shortcuts, we check to see if a job is selected _before_ triggering the event, rather than on the listeners side. If there's a case for doing the latter then I'm all ears - but it seems like this should be doing the former otherwise, for consistencies sake? :-)